hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/TimelineServerBasedWriteMarkers.java

```diff
@@ -86,6 +89,17 @@ public TimelineServerBasedWriteMarkers(HoodieTable table, String instantTime) {
     this.timeoutSecs = timeoutSecs;
   }
 
+  @Override
+  protected Path getMarkerPath(String partitionPath, String dataFileName, IOType type) {
+    return new Path(partitionPath, getMarkerFileName(dataFileName, type));
+  }
```

Comment:
Why override the partition path as relative?

Reply:
Timeline server based markers store relative paths of markers.
cat <basepath>/.hoodie/.temp/<ts>/markers.type
TIMELINE_SERVED_BASED

cat <basepath>/.hoodie/.temp/<ts>/MARKERS<N>
2020/06/01/file1.marker.MERGE

It was added to address some test failures is my recollection.

Reply:
Removed.